export_dagmc_cmd/iGeom_test.cpp

```diff
@@ -0,0 +1,93 @@
+#include "iGeom_test.hpp"
+#include "iGeom_funcs.hpp"
+#include <vector>
+#include "CubitInterface.hpp"
+
+// CGM includes
+#include "GeometryModifyTool.hpp"
+#include "Body.hpp"
+#include "RefEntity.hpp"
+#include <iostream>
+
+iGeom_test::iGeom_test()
+{
+  //default values
+  radius = 2.0;
+  radius2 = 1.0;
+}
+
+iGeom_test::~iGeom_test()
+{}
+
+std::vector<std::string> iGeom_test::get_syntax()
+{
+  // Define the syntax for the command. Note the syntax is a modified BNF
+  // format. Full documentation on the command specification syntax can be
+  // found in the documentation.
+  std::string syntax =
+      "iGeom_test"
+      "[<value:label='radius',help='<radius>'> ]"
+      "[<value:label='radius2',help='<radius2>'>]";
+
+  std::vector<std::string> syntax_list;
+  syntax_list.push_back(syntax);
+
+  return syntax_list;
+}
+
+std::vector<std::string> iGeom_test::get_syntax_help()
+{
+  std::vector<std::string> help;
+  return help;
+}
+
+std::vector<std::string> iGeom_test::get_help()
+{
+  std::vector<std::string> help;
+  return help;
+}
+
+bool iGeom_test::execute(CubitCommandData &data)
```

Comment:
Some of these tests are commented... can they go? Or are they not working for us right now?